### PR TITLE
Added remote backup script.

### PR DESCRIPTION
**Pull request by coreygoettsch**

### CODE REVIEW

File: remotebackup.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+#This script connects  to a remote server via ssh, creates a .tar.gz backup, and uses scp to secure copy it to the host computer.
```
Comment (benjwg) [inline remotebackup.sh]: Apply my previous comment about comments to this.

Comment (coreygoettsch) [reply remotebackup.sh]: It's because I'm too verbose in  the comment, right?  I'll simplify the language.  Thanks!

Comment (benjwg) [reply remotebackup.sh]: Look at the suggested format I gave in the other comment and follow that - more standard and more legible

Comment (coreygoettsch) [reply remotebackup.sh]: I found the suggested format  in the comment, and  I'll make the necessary changes.  Thank you.

File: remotebackup.sh [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
 
+#Example  use:
+#./remotebackup.sh username remoteip /directory/to/backup nameofbackupfile
+
+# Arguments 
+# $1 username of remote user
+# $2 ip address of remote server
+# $3 remote directory to be backed up (e.g. /var/www/bob)
+# $4 is the name of the .tar.gz backup (e.g. bobbackup) 
+# End of arguments
```
Comment (benjwg) [inline remotebackup.sh]: much better - hate to nit but i dont _really_ think you need all this `start` and `end `stuff` - or can you point me to something that shows this is conventional for shell scripting?

Comment (coreygoettsch) [reply remotebackup.sh]: I don't think it's conventional.  The guy who teaches my online course  about shell scripting does it so  I just  did it too.  If you think it's superfluous, I'll get rid of it.

**Comment by benjwg** [comment]
Yea I say get rid of it

**Pull request merged**